backend/src/User.ts

```diff
@@ -80,6 +81,7 @@ export class User {
 
     Logger.info('SIGN OUT USER', { user: this })
 
+    EventBus.emit(ConnectionPool.EVENTS.updated, [])
```

Comment:
@JamieRuderman do you mean that we should remove this line?

Reply:
Yes remove and add to the `clearMemory` method.